### ISSUE TITLE
Bug: Lifeline deactivation using -- not working

### ISSUE THREAD

**Issue by chipbite** [plantuml/plantuml#1048]
Hello!

Lifeline activation is not removed, it seems. Not sure if I am doing something wrong, see example:

```plantuml 
@startuml
!pragma teoz true
scale 1600 width
hide footbox
hide unlinked 

actor Customer as cust
boundary "UI" as ui
control "EntityManager" as entityMgr
entity "Core Entity" as anEntity

ui -> anEntity++: Create

cust -[#navy]-\ ui: Drag
& ui -> entityMgr: add
& entityMgr -> anEntity++: (...)

cust -[#navy]-\ ui: End
& ui -> entityMgr: remove
& entityMgr -> anEntity--: (...)
entityMgr -> cust: uh-huh
cust -> entityMgr: oops


@enduml
```

### Results in: 
![image](https://user-images.githubusercontent.com/559199/174799105-326faa1d-2d16-4a04-b545-08b887bc61d2.png)

### Currently rendered image from server: 

![](https://www.plantuml.com/plantuml/png/TP6nJiD038PtFuK94W9LLvM58GEg53emC381WvipoOdaN3c-GdXwcfw8L5Gs_vzztp_SUZ1IItq75pkbfYSqbcyqJGn3JHtZxTrgXP_1Mmjjy8pl8hQLhnnIx4ByO8y0L9ieLcamwLcH1gmFCsmbHKywOl7yM4mq1QWbcag7nIPQiF698ZMinordqYZa2OjAb34V7YyevW2G0hhxEIyM9LRAP0mmDQDxkOoq7z_SAwPGueDI0rUO7yqr9PBt1po3lyhhvN9vSzwuYVwSKBcNFV_dTExNURAU_2McrhMfpMMdKf7T001hZdxwh1y0)

# Workaround 
is to explicitly use `deactivate`, which works well here.

**Comment by jhoehle** [comment]
You are doing it wrong indeed. In `entityMgr -> anEntity --`
the -- deactivates the sender, not the receiver. The notation may be confusing. Explicit `Deactivate` is what you want.
IMHO, this bug report is invalid; somebody please close it.

**Comment by chipbite** [comment]
@jhoehle : 
> deactivates the sender, not the receiver.

You have understood the syntax correctly.

TLDR your reply: The workaround I stated originally of using explicit is the only option, currently.

----

The use case is very odd, so I am closing this.
